pkg/kubectl/cmd/util/factory_test.go

```diff
@@ -37,6 +37,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
Try temporarily removing the `2>/dev/null` from line 540 of `hack/lib/util.sh` and see if it prints any useful error messages.